src/Bonnier/WP/Cache/Models/Post.php

```diff
@@ -13,6 +13,8 @@ class Post
     // if they ever are.
     const ACF_CATEGORY_ID = 'field_58e39a7118284';
 
+    private static $postTypes = ['contenthub_composite', 'post'];
```

Comment:
Why not make this a constant, if it's not supposed to be changed?

Reply:
For now they are not supposed to change. I can make them a const if you like? 👍